tests/kuttl/tests/galera-3replicas/02-cleanup.yaml

```diff
@@ -4,3 +4,9 @@ delete:
 - apiVersion: core.openstack.org/v1beta1
   kind: OpenStackControlPlane
   name: openstack
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    oc delete -n $NAMESPACE pvc mysql-db-openstack-galera-0 mysql-db-openstack-galera-1 mysql-db-openstack-galera-2
```

Comment:
I know it's probably impossible for this situation to occur, but what it makes sense to include `--ignore-not-found` just in case these are somehow missing?

Reply:
It's unlikely, but it won't hurt, so I added just in case